distribution/docs/src/main/resources/content/_securityFramework/federated-identity.adoc

```diff
@@ -0,0 +1,76 @@
+:title: Federated Identity
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:order: 10
+:summary: How a user's identity is shared with federated Sources during queries
```

Comment:
✏️ Looking through our docs I think this should either be `federated sources` or `Federated Sources`.

Reply:
I'd say `federated sources` unless you're talking about a specific `XYZ Federated Source`

Reply:
I used this capitalization to emphasize that I'm referring to the DDF concept of a source, not using the term generically.